wallet/udb/addressmanager.go

```diff
@@ -854,10 +869,10 @@ func (m *Manager) AddrAccount(ns walletdb.ReadBucket, address dcrutil.Address) (
 // passphrase may be used to bump the computational difficulty needed to brute
 // force the passphrase.
 func (m *Manager) ChangePassphrase(ns walletdb.ReadWriteBucket, oldPassphrase, newPassphrase []byte,
-	private bool, config *ScryptOptions) error {
```

Comment:
Yes. There was no way for the user to change it from the config.  If we bump the parameters, then they will still take effect next passphrase change.  The tests still use dumbed down values so they run faster.